getting-started/templates/systemlink-secrets.yaml

```diff
@@ -214,6 +214,18 @@ dataframeservice:
       ## The password for the Dremio instance.
       ##
       password: ""  # <ATTENTION>
+  sldremio:
```

Comment:
Note that this section is a bit unique in this file. We aren't deploying a secret, we're just taking advantage of the fact that this is an encrypted values file where we can store hardcoded credentials.

Reply:
Wait, so how is this data stored then? Are these credentials in the clear on the cluster? That seems bad.

Reply:
That's a good point. They end up in a ConfigMap that gets mounted as a volume in the Dremio pod.

Reply:
It looks like it goes into a configmap ("dataframeservice-dfs-dremio-config" on stratus-dev, if you're curious). The chart doesn't support referencing these values from a secret, so we would need to add that functionality ourselves. I can go ahead and make us a story to do that.

Reply:
I made a story [here](https://dev.azure.com/ni/DevCentral/_workitems/edit/2107157).